Client/game_sa/CHudSA.cpp

```diff
@@ -43,6 +43,8 @@ void CHudSA::Disable(bool bDisabled)
     else
         MemPut<BYTE>(FUNC_Draw, 0x80);
 
+    SetComponentVisible(HUD_ALL, !bDisabled);    
```

Comment:
Does it work when HUD is partially hidden?
1. Disable 'money' component
2. showhud -> all hud is disabled
3. showhud again -> ?

Reply:
it will show all components. Will fix that